src/PerformanceTests/Tests/Statistics.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
```

Comment:
Need to read stuff off Permutation object. So this can go where that is. There was hardly any benefit of having a project with just Statistics in it.